app/views/content/life-as-a-teacher/explore-subjects/geography.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Become a geography teacher
+subject: geography
+title_paragraph: |-
+  <p>
+  In teaching geography you’ll guide students through the fascinating diversity of the Earth. You’ll inspire them to reflect on the world they live in and discover their unique place within it.</p>
+  <p>
+  Tax-free bursaries of $bursaries_postgraduate_geography$ are available for eligible trainee geography teachers.</p>
+description: |-
+    Explore what's involved in becoming a geography teacher, including what you'll be teaching and what funding is available to help you train.
+layout: "layouts/minimal"
+colour: pastel yellow-yellow
+image: "static/images/content/hero-images/languages.jpg"
```

Comment:
We do have geography images - there are a couple of variations of this teacher:

![image](https://github.com/user-attachments/assets/15f35bb2-ade0-449e-906c-ca8a718a9241)

@jenhadfield-dfe can help you identify which images you can use (or can help upload them if we haven't done this already)